frontend/hooks/me/index.ts

```diff
@@ -2,20 +2,23 @@ import { useMemo } from 'react';
 
 import { useQuery } from 'react-query';
 
+import { Queries } from 'enums';
 import { User } from 'types/user';
 
 import API from 'services/api';
 
 export default function useMe() {
   const query = useQuery<User>(
-    'user',
+    Queries.User,
     () =>
       API.request({
         method: 'GET',
         url: '/api/v1/user',
       }).then((response) => response.data.data),
     {
-      retry: 1,
+      retry: 0,
+      staleTime: Infinity,
+      refetchOnMount: false,
```

Comment:
I don't think an infinite stale time is a good idea. It could block the UI.

I would either use the default values (provided by React Query), or set a default for all the queries, or document here why we use different values.